### ISSUE TITLE
Redirect instead of proxy could reduce request time period

### ISSUE THREAD

**Issue by glentregoning** [taskrabbit/sailthru-proxy#1]
Firstly, neat project! 

However on looking over your implementation I notice that you proxy the request to Sailthru. I was thinking given the proxying is only used for non iOS clients or iOS clients without the app wouldn't a redirect be sufficient for everything except static assets and the /apple-app-site-association.

**Comment by evantahler** [comment]
Makes sense! That should be simple enough to determine based on the headers.  Pull requests welcome!
